exchanges/huobi/huobi_wrapper.go

```diff
@@ -999,11 +999,30 @@ func (h *HUOBI) SubmitOrder(ctx context.Context, s *order.Submit) (*order.Submit
 			oDirection = "SELL"
 		}
 		var oType string
-		switch {
-		case s.Type == order.Limit:
+		switch s.Type {
+		case order.Market:
+			// https://huobiapi.github.io/docs/dm/v1/en/#order-and-trade
+			// At present, Huobi Futures does not support market price when placing an order.
```

Comment:
🐒 I was struggling with what this truly meant. Perhaps this might help:
```suggestion
			// At present, Huobi Futures does not support unlimited slippage market price when placing an order.
```

Sorry, but I have to check: Are we really sure that BBO *isn't* a normal market order, though?
I'm reading the docs and it kinda seems like BBO is a badly worded full-slippage-market order, whereas the optimal-N are limited slippage.

Reply:
Outstanding question: Are we sure BBO isn't just a normal market order?

Reply:
I missed the last line of the docs.
> place a limit order for this price.

Outstanding question resolved, thread resovled.